dev/tests/static/framework/Magento/Sniffs/Annotation/MethodArgumentsSniff.php

```diff
@@ -663,6 +663,10 @@ private function noneParamsAligned(array $argumentPositions, array $commentPosit
         foreach ($argumentPositions as $index => $argumentPosition) {
             $commentPosition = $commentPositions[$index];
             $type = $paramDefinitions[$index]['type'];
+            if ($type == null) {
```

Comment:
Why need this?

Reply:
@orlangur Found one new issue 
![DeepinScreenshot_select-area_20190520153623](https://user-images.githubusercontent.com/2481206/58021964-0f231d80-7b15-11e9-9bc7-1c025b43d70e.png)

Reply:
Could you add more info - I'm more suspicious about the `unset`

Reply:
@orlangur well, unset is not needed, when we have type null, we can just skip iteration.

Reply:
Ok, please do so and also make a check strict (`=== null`).